WordPressKit/WordPressKit/Activity.swift

```diff
@@ -96,28 +100,34 @@ public struct RemoteActivityObject {
 
 enum RemoteActivityError: Error {
     case missingActivityId
+    case missingPublishedDate
+    case incorrectPusblishedDateFormat
+}
+
+public struct ActivityName {
+    public static let fullBackup = "rewind__backup_complete_full"
 }
 
-extension RemoteActivity: CustomDebugStringConvertible {
+extension Activity: CustomDebugStringConvertible {
     public var debugDescription: String {
         let dateFormatter = ISO8601DateFormatter()
-        let publishedDate = published != nil ? dateFormatter.string(from: published!) : ""
-        return "<RemoteActivity: (activityID: \(activityID), summary: \(summary), name: \(name), type: \(type) " +
-               "gridicon: \(gridicon), status: \(status), rewindable: \(rewindable), published: \(publishedDate) " +
-               "actor: \(actor.debugDescription), object: \(object.debugDescription), " +
-               "target: \(target.debugDescription), items: \(items != nil ? items.debugDescription : "[]")>";
+        return "<Activity: (activityID: \(activityID), summary: \(summary), name: \(name), type: \(type) " +
+               "gridicon: \(gridicon), status: \(status), rewindable: \(rewindable), " +
+               "published: \(dateFormatter.string(from: published)) actor: \(actor.debugDescription), " +
+               "object: \(object.debugDescription), target: \(target.debugDescription), " +
+               "items: \(items != nil ? items.debugDescription : "[]")>";
```

Comment:
While testing this I realized that lldb already shows all the struct properties, so I don't think we need the custom descriptions